quickstarts/gradle/spring-boot/build.gradle

```diff
@@ -9,6 +9,10 @@ plugins {
 group = 'org.eclipse.jkube.quickstarts.gradle'
 version = '0.0.1-SNAPSHOT'
 sourceCompatibility = '11'
+description = """\
+    Spring Boot Web application with a single @RestController.
+    Shows how to deploy Spring Boot applications to Kubernetes and OpenShift using Eclipse JKube.
+"""
```

Comment:
this shouldn't be applicable for this PR. Is this a mistake? Is there any other reason why you added it=

Reply:
This looks wrong. Let me update it.